packages/drivers/redux-little-router/src/reduxLittleRouter.js

```diff
@@ -0,0 +1,32 @@
+import { push, replace, go, goBack, goForward, block, unblock } from 'redux-little-router'
+
+const getParentResultParam = (result, key) => {
+  if (!result) return undefined
+  if (result[key]) return result[key]
+  return getParentResultParam(result.parent, key)
+}
+
+const isParentResultParam = (result, key, value) => {
+  if (!result) return false
+  if (result[key] === value) return true
+  return isParentResultParam(result.parent, key, value)
+}
+
+export default selector => ({ dispatch, getState }) => ({
+  /* actions */
+  push: path => dispatch(push(path)),
+  replace: path => dispatch(replace(path)),
+  go: nbLocations => dispatch(go(nbLocations)),
+  goBack: () => dispatch(goBack()),
+  goForward: () => dispatch(goForward()),
+  block: callback => dispatch(block(callback)),
+  unblock: () => dispatch(unblock()),
+  /* route selectors */
+  getState: () => selector(getState()),
```

Comment:
You can use constant to create fonction and use them so this is easier to maintain (see next comment in the file)

Reply:
I hope that I made it as you think it